tests/cli/workflow/nf_analysis/test_cli_config_case.py

```diff
@@ -24,7 +24,7 @@
 
 @pytest.mark.parametrize(
     "workflow",
-    NEXTFLOW_WORKFLOWS,
+    NEXTFLOW_WORKFLOWS + [Workflow.NALLO],
```

Comment:
I guess this is temporary, to avoid breaking other parametrized tests?

Reply:
Yes, correct. If I add it to NEXTFLOW_WORKFLOWS, it tests all nf-tests, including e.g. metrics-deliver.